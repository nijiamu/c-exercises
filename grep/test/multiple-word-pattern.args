"no newline" test.txt test2.txt
