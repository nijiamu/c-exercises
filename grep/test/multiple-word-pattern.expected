test.txt:4:last line has no newline
